src/pages/code-reviews-getting-started/index.md

```diff
@@ -0,0 +1,88 @@
+
+## Code reviews in practice
+
+Sounds great on paper, right? 🌈
+
+### Convince the boss
+
+In practice, the first step is to “convince the boss”. Show them those great [metrics on the effectiveness of code reviews](https://blog.codinghorror.com/code-reviews-just-do-it/). Talk about how reviews improve [quality _in terms of business value_](https://www.codemate.com/quality-in-terms-of-business-value). Their main scepticism might be in the cost of having more people working on an otherwise identical amount of changes – this is a good time to talk about the economics of fixing software defects:
+
+![Bar chart showing the relative cost of fixing software defects in each phase of the software development lifecycle. The relative cost increases from 1 in design, to 6.5 in implementation, 15 in testing, and 100 in maintenance.](./images/relative-cost-of-fixing-defects-sdlc.png)
+
+_Relative cost of fixing defects in the software development lifecycle. The later, the worse. Source: IBM System Science Institute._
+
+### Get started with pull request reviews
+
+With the business case taken care of, simply try code reviews for yourself! Every major Git platform has a pull request feature with built-in [review workflows](https://help.github.com/articles/requesting-a-pull-request-review/). If yours doesn’t, you can try doing code reviews in person instead, [pair programming style](https://en.wikipedia.org/wiki/Extreme_programming). No matter how you do it – start small, and see how it works out for you.
+
+## Resources to get started
+
+To give you a headstart in implementing code reviews for your teams, here are three things you will need:
+
+1.  [Code review guidelines](https://github.com/CodemateLtd/cookbook/blob/master/CONTRIBUTING.md). We usually put those in a `CONTRIBUTING.md` file at the root of the project.
+2.  [A pull request template](https://github.com/CodemateLtd/cookbook/blob/master/.github/PULL_REQUEST_TEMPLATE.md). This is where you state that the team does code reviews, that they will be enforced, and that pull requests should be authored accordingly. Ours is simple but gets the job done.
+3.  [Useful resources about how to do this well](https://github.com/CodemateLtd/cookbook/blob/master/CONTRIBUTING.md). Code reviews are a solved problem, there are great resources out there to take inspiration from.
+
+Take all of these, sprinkle some LGTM ([Looks Good To Me](http://knowyourmeme.com/memes/lgtm)) images on top when approving changes, and you’re good to go. Code reviews can feel tense at times, those silly images set a more relaxed tone, and can ease the relationship between reviewers and reviewees. Plus, a nice picture is a great reward for good work!
```

Comment:
This is important even though looking for some nice pics in the middle of the busy things might feel tedious.

Another thing to preserve humane and friendly atmosphere I feel should be mentioned somewhere could go along the lines:
> Be kind and let tooling do the nitpicking. Choose common formatting and enforce it via tooling (Prettier, linters etc). Let humans focus on logic, use cases, readability and learning.
>
> Common formatting makes diffs easier to read and you don't really want to discuss tabs vs spaces on each review.

Reply:
👌 good point. I'll try to cover this around this "LGTM" part.